components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/add-hardware-account-modal.tsx

```diff
@@ -57,12 +62,12 @@ export const AddHardwareAccountModal = () => {
   }, [createAccountOptions, accountTypeName])
 
   // methods
-  const setImportError = React.useCallback((hasError: boolean) => {
+  const setImportError = React.useCallback((hasError: ImportAccountErrorType) => {
     dispatch(WalletPageActions.setImportAccountError(hasError))
   }, [])
 
   const closeModal = React.useCallback(() => {
-    setImportError(false)
+    setImportError(undefined)
```

Comment:
setImportError is missing from dep array.

Reply:
Added!